# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wavemon/linux-header.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- wavemon-0.4.0b/iw_if.h.vanilla	2007-05-12 11:06:21.003361944 +0200
+++ wavemon-0.4.0b/iw_if.h	2007-05-12 11:07:03.401778092 +0200
@@ -19,6 +19,8 @@
  */
 
 #include <netdb.h>
+#include <linux/types.h>
+#include <linux/if.h>
 #include <linux/wireless.h>
 
 #define IW_STACKSIZE 1024
